tsconfig.base.json

```diff
@@ -15,8 +15,8 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "qwik-nx": ["packages/qwik-nx/src/index.ts"],
-      "@qwikifiers/e2e/utils": ["e2e/utils"]
```

Comment:
`pnpm nx format` did it 🤠

Reply:
Yeah, not sure how this sneaker in, we have format in pre-commit 😅